spinta/manifests/xsd/helpers.py

```diff
@@ -855,10 +862,31 @@ def _add_refs_for_backrefs(self):
                     referenced_model = self.models[prop["model"]]
                     referenced_model.add_ref_property(model)
 
-    def _sort_properties_alpabetically(self):
+    def _sort_properties_alphabetically(self):
         for model in self.models.values():
             model.properties = dict(sorted(model.properties.items()))
 
+    def _remove_duplicate_models(self):
+        """removes models that are exactly the same"""
+
+        model_pairs = {}
+
+        for model_name, model in self.models.items():
+            for another_model_name, another_model in self.models.items():
```

Comment:
Is it only possible for 2 models to be identical and not 3 or more?

Reply:
it is possible, yes. And this should catch them all. Or at least I think so. 

`model` is the model that it's being compared against, and `another_model` is the one that's being compared. There might be many cases of `another_model` being equal to `model` and it's added to the `model_pairs" every time. I used `another_model_name` as a key for this reason. So if we have model `Person`, then an identical model `Person1` and `Person2`, the model_pairs will look like this: `{"Person1": "Person", "Person2": "Person"}` and then the references will be reassigned for each of those models.

Reply:
> Is it only possible for 2 models to be identical and not 3 or more?

I can be wrong though, if you think that it won't remove more than one duplicate, could you suggest what I should do differently?

Reply:
You're right, my mistake!